doc/cephadm/services/index.rst

```diff
@@ -81,6 +81,13 @@ system name:
 
     ceph orch ps --daemon_type osd --daemon_id 0
 
+.. note::
+   The information shown by ``ceph orch ps`` may not be up to date. Daemon status is refreshed every 10 minutes by default.
+   This can be changed by setting the configuration variable ``mgr/cephadm/daemon_cache_timeout`` (in seconds) e.g:
+   ``ceph config set mgr mgr/cephadm/daemon_cache_timeout 60`` will reduce the refresh interval to one minute. In general
+   ``REFRESHED`` will indicate how old the information shown by ``ceph orch ps`` and similar commands is. Normally the information
+   is referesh every ``daemon_cache_timeout`` unless the ``--refresh`` option is provided.
```

Comment:
Sorry, seem to have missed in initial review
```suggestion
   is refreshed every ``daemon_cache_timeout`` seconds unless the ``--refresh`` option is provided.
```
Also, do we want to add a little qualifier about what `--refresh` actually does. Something like "Note that while the option will tell cephadm to begin a refresh of the information, it does not mean the output received from the command will be up to date". Not sure if that's redundant given the first sentence you added. Wdyt?

Reply:
> 

No prob, I rephrased the whole doc seeking for more clarity. Please, have a look and let me know if you have any comments/suggestions. Thanks.